spot_wrapper/wrapper.py

```diff
@@ -1127,48 +2163,142 @@ def _match_edge(self, current_edges, waypoint1, waypoint2):
+                        quality_percent=50,
+                    )
+                ]
+            )[0]
         except UnsupportedPixelFormatRequestedError as e:
             return None
 
     def get_frontright_rgb_image(self):
         try:
-            return self._image_client.get_image([build_image_request(
-                "frontright_fisheye_image", pixel_format=image_pb2.Image.PIXEL_FORMAT_RGB_U8, quality_percent=50)])[0]
+            return self._image_client.get_image(
+                [
+                    build_image_request(
+                        "frontright_fisheye_image",
+                        pixel_format=image_pb2.Image.PIXEL_FORMAT_RGB_U8,
+                        quality_percent=50,
+                    )
+                ]
+            )[0]
         except UnsupportedPixelFormatRequestedError as e:
             return None
 
     def get_left_rgb_image(self):
         try:
-            return self._image_client.get_image([build_image_request(
-                "left_fisheye_image", pixel_format=image_pb2.Image.PIXEL_FORMAT_RGB_U8, quality_percent=50)])[0]
+            return self._image_client.get_image(
+                [
+                    build_image_request(
+                        "left_fisheye_image",
+                        pixel_format=image_pb2.Image.PIXEL_FORMAT_RGB_U8,
+                        quality_percent=50,
+                    )
+                ]
+            )[0]
         except UnsupportedPixelFormatRequestedError as e:
             return None
 
     def get_right_rgb_image(self):
         try:
-            return self._image_client.get_image([build_image_request(
-                "right_fisheye_image", pixel_format=image_pb2.Image.PIXEL_FORMAT_RGB_U8, quality_percent=50)])[0]
+            return self._image_client.get_image(
+                [
+                    build_image_request(
+                        "right_fisheye_image",
+                        pixel_format=image_pb2.Image.PIXEL_FORMAT_RGB_U8,
+                        quality_percent=50,
+                    )
+                ]
+            )[0]
         except UnsupportedPixelFormatRequestedError as e:
             return None
 
     def get_back_rgb_image(self):
         try:
-            return self._image_client.get_image([build_image_request(
-                "back_fisheye_image", pixel_format=image_pb2.Image.PIXEL_FORMAT_RGB_U8, quality_percent=50)])[0]
+            return self._image_client.get_image(
+                [
+                    build_image_request(
+                        "back_fisheye_image",
+                        pixel_format=image_pb2.Image.PIXEL_FORMAT_RGB_U8,
+                        quality_percent=50,
+                    )
+                ]
+            )[0]
         except UnsupportedPixelFormatRequestedError as e:
             return None
 
-    def get_images(self, image_requests: list[image_pb2.ImageRequest]) -> Optional[ImageBundle]:
+    def get_images(
```

Comment:
Is your team using 80char for your style guide? We are using 120 which is why this wasn't broken onto multiple lines.

Reply:
I've instituted the use of the default black formatter https://black.readthedocs.io/en/stable/index.html for the ROS1 repository. The default line length is 88 (https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length).